src/solidity-frontend/solidity_grammar.cpp

```diff
@@ -156,6 +156,10 @@ TypeNameT get_type_name_t(const nlohmann::json &type_name)
     {
       return TupleTypeName;
     }
+    if (typeIdentifier.substr(0, 10) == "t_mapping(")
```

Comment:
Shall we use `std::string::compare` function instead of `substr` for more efficient comparison?

Reply:
Will do. Thanks for the suggestion.